src/main/java/training/subsystems/wrist/WristConstant.java

```diff
@@ -0,0 +1,10 @@
+package training.subsystems.wrist;
+
+public class WristConstant {
+
+	public static final int MOTOR_ID = 11;
+	public static final int MOTOR_P = 1;
+	public static final int MOTOR_I = 0;
+	public static final int MOTOR_D = 0;
```

Comment:
no need to put "MOTOR" before
![image](https://github.com/user-attachments/assets/d1ae29a8-28f2-4e05-84fb-a51f682fee78)